themes/mlibrary_new/items/show.php

```diff
@@ -1,5 +1,63 @@
 <?php
-fire_plugin_hook('public_items_show', array('view' => $this, 'item' => $item));
+ /**
+  * Copyright (c) 2016, Regents of the University of Michigan.
+  * All rights reserved. See LICENSE.txt for details.
+  */
+  $item_title = strip_formatting(metadata('item', array('Dublin Core', 'Title')));
+  if (empty($item_title)) { $item_title = __('[Untitled]'); }
+
+  echo head(
+    array(
+      'title'     => $item_title,
+      'bodyid'    => 'items',
+      'bodyclass' => 'show item'
+    )
+  );
+ // if the item is part of the exhibit builder layout
+ if (isset($_GET['exhibit']) && isset($_GET['page'])) {
+    $pageClean = html_escape($_GET['page']);
+    $exhibitClean = html_escape($_GET['exhibit']);
+    echo '<div class="exhibit-item-back-button"><a href="' .
+        exhibit_builder_exhibit_uri(
+          get_record_by_id('exhibit', $exhibitClean),
+          get_record_by_id('exhibit_page', $pageClean)
+        ) .
+      '">Return to Exhibit</a></div>';
+  }
+?>
+<h1><?php echo $item_title; ?></h1>
+<div id="primary">
+  <?php
+    $index_file = 0;
+    $audio_file = false;
+    $item_type = (empty($item->getItemType()->name)) ? 'Image' : $item->getItemType()->name;
+    $theme_name = (isset($exhibit->theme)) ? $exhibit->theme : 'mlibrary_new';
+
+    if ($item_type != 'Video') { ?>
+       <div id="item-images">
+<button id="action-zoom-in">Zoom In</button>
+<span id="span-zoom-status"></span>
+<button id="action-zoom-out">Zoom Out</button>
+<button id="action-reset-viewer">Reset View</button>
+<button id="action-rotate-left">Rotate left</button>
+<button id="action-rotate-right">Rotate Right</button>
+         <?php
+               $file = $item->getFiles();
+              $filePath = FILES_DIR.'/'.$file[0]->getStoragePath('original');?>
+              <div id="fsize_images">
+             <!-- <div id="image-zoomer-os"  data-identifier='https://bertrama.www.lib.umich.edu/online-exhibits-2.3/iiif/002b7e2b8a64911b76bb8d35535e39ed.jpg/info.json' 
+                   style="height: 600px; width: 100%;"></div> -->         
+             <div id = "image-zoomer-os"  data-identifier = 'https://quod.lib.umich.edu/cgi/i/image/api/tile/sclaudubon:B6719890:29376_0019/info.json' style = 'height: 600px; width: 100%;'></div> 
+            </div> <!--fsize_images-->
+    <?php echo '</div>'; //item-images
```

Comment:
We can move the div closing from L52 to L51 and leave it as markup instead of a string printed by php.